### PR TITLE
release: 450.57

### PR DESCRIPTION
**Pull request by mmstick**

### CODE REVIEW

File: debian/changelog [hunk 1]
```diff
@@ -1,8 +1,8 @@
-nvidia-graphics-drivers-440 (440.100-1pop1) focal; urgency=medium
+nvidia-graphics-drivers-450 (450.57-1pop1) focal; urgency=medium
 
-  * Upstream release 
+  * Beta
```
Comment (jackpot51) [inline debian/changelog:3]: 450.57 is no longer a beta, so this should say Upstream release like the others

**Comment by leviport** [comment]
I've checked thelio-b2 with a 1070 and major-r2 with a 2080 and they both look good. Discrete cards in desktops seem hunkydory.

**Comment by leviport** [comment]
I'm seeing an interesting issue with hybrid graphics on the oryp6. When I'm in Nvidia mode with my 4K display stacked on top of my built-in display, everything works as expected: 
Screenshot from 2020-07-14 07-37-19

However when I'm in hybrid graphics, it almost seems like the two displays are joined into a single display, even though they are different resolutions. In this screenshot, you can see my top bar. Everything outside the edges of my top bar on the bottom built-in display is cut off. The system menu appears halfway off the edge of the built-in display.
Screenshot from 2020-07-14 07-42-01

Also when I enter the activities overview in this state, the windows stay on top of everything on the activities overview so I can't see any of my search bar, search results, dash application lists, or workspace switcher. It's hard to tell, but I have the application list open behind everything:
Screenshot from 2020-07-14 07-41-48

Since showing all applications in overview hides open windows only on the primary display, I'm pretty sure it's treating the internal display as an extension of the external 4K display.

FWIW, things seem normal if I arrange the external display to the side of the built-in display rather than above it.

**Comment by kellyhair** [comment]
What's the ETA on this dropping? 

I have an oryp6 and the 440.100 driver is not working for a Steam game I want to play. The game does load in `integrated graphics` mode but it's unplayable. 

Also, is there a `beta` repo I can add? Downloading the Nvidia driver (  NVIDIA-Linux-x86_64-450.57 ) warned that installation may screw up the quick switching..

**Comment by leviport** [comment]
> What's the ETA on this dropping?

When Nvidia releases a version that doesn't break Mutter. Since it's a proprietary driver, there isn't much we can do besides wait for a new release.

**Comment by kellyhair** [comment]
Looks like there's a new release in the `long live branch` -> [450.66](https://www.nvidia.com/Download/driverResults.aspx/163238/en-us)

Is there an issue to track the updated version?

**Comment by leviport** [comment]
https://github.com/pop-os/gnome-shell/pull/40 is being tested currently. If it doesn't have any regressions, it will probably go out in the next few days. At the very least, it installed on an Oryx without messing anything up so far.

**Pull request closed without merging**